index.html

```diff
@@ -4,17 +4,289 @@
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="./pages/about.html">About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="./pages/services.html">Services</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="./pages/blog.html">Blog</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="./pages/contact.html">Contact</a>
+                        </li>
+                    </ul>
+                </div>
+                <h1 class="display-sm col-12 col-md-5 offset-md-1 align-self-baseline">Maria Hope, Ph.D. - Psychologist</h1>
+            </div>
+            
+            
 
+                <!-- APPOINTMENT BTN MD-LG DISPLAY -->
+            <div class="display-md col-md-10 justify-content-end">
+                <button class="appointment-btn align-self-end ">Book an Appointment</button>
+            </div>
+
+                <div class="display-md col-md-12">
+                    <div class="row flex-grow-1">
+                            <ul class="navbar-nav display-order col-md-5 justify-content-end ">
+                                <li class="nav-item">
+                                    <a class="nav-link active" aria-current="page" href="#"> Home</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="./pages/about.html">About</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="./pages/services.html">Services</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="./pages/blog.html">Blog</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="./pages/contact.html">Contact</a>
+                                </li>
+                            </ul>
+                    
+                        <h1 class="padding-top col-md-5 offset-md-1 align-self-baseline">Maria Hope, Ph.D. - Psychologist</h1>
+                    </div>
+                </div>
+                
+            
+        </nav> 
+    
+    </header>
+    <main class="text-center container">
+        <section class="hero row">
+            <div class="hero-md-img p-0 display-order col-md-6">
+            </div>
+                <img class="hero-img p-0 col-12 display-sm img-fluid" src="./img/erol-ahmed-aIYFR0vbADk-unsplash.jpg" alt="Succulent plant">
```

Comment:
You shouldn't have to include the succulent image twice here. Flexbox's `order` rule should be enough? Tip if it's not working, you need to apply the order to both elements, not just the image. 

Also, avoid using `col-...` classes on images directly. Set the col on a div first, and put your image inside.

Reply:
100% agree don't know why I did that must of been rushing or it did register at the time. adding 'col..' to images isn't advised, I used it because it place it perfectly for how I wanted it to be displayed and to achieve that I may of had to use 'object-fit', which again not advised because of IE not being compatible. What would be the best practice for placing images with bootstrap? Or is simply adding 'col..' to the div going to achieve that? i will attempt it myself but if there is a better way of doing so that would be great to know and add to my arsenal.